### PR TITLE
Regarding Issue#3

### PR DESCRIPTION
**Pull request by Tawishi**
This is what I have done so far. Please review.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,8 +1,39 @@
...
+
+![git](https://github.com/Tawishi/game_of_life/blob/Issue%233/2.jpg)
+
+   -> Clone URI.
+
+![git_clone](https://github.com/Tawishi/game_of_life/blob/Issue%233/3.jpg)
+
+2. Set URI GIT_CLONE_URI.
+
+![Import_project](https://github.com/Tawishi/game_of_life/blob/Issue%233/issue2.jpg)
```
Comment (48cfu) [inline README.md:29]: set links to point to original repository

File: README.md [hunk 2]
```diff
@@ -1,8 +1,39 @@
...
+ - [Langton’s Ant](https://en.wikipedia.org/wiki/Brian%27s_Brain)
+ - [Brian’s Brain](https://en.wikipedia.org/wiki/Brian%27s_Brain)
+ - [Day and Night](https://en.wikipedia.org/wiki/Day_and_Night_(cellular_automaton))
+ - [Pretty print](https://chrome.google.com/webstore/detail/prettyprint/nipdlgebaanapcphbcidpmmmkcecpkhg?hl=en) 
+ - etc...
+ 
+To build current code use [Eclipse IDE for C/C++ Developers](https://www.eclipse.org/downloads/packages/release/helios/sr2/eclipse-ide-cc-developers).
+The instructions in Eclipse are:
+
+**(please delete .cproject and .project from your branch before importing it to eclipse. If possible add them to .gitignore)**
```
Comment (48cfu) [inline README.md]: just delete this line. not necessary anymore

File: README.md [hunk 3]
```diff
@@ -1,8 +1,39 @@
...
+# Game Of Life 
+[Convey's game of life](https://en.wikipedia.org/wiki/Conway%27s_Game_of_Life) is a zero-player game, meaning that its evolution is determined by its initial state, requiring no further input. One interacts with the Game of Life by creating an initial configuration and observing how it evolves, or, for advanced players, by creating patterns with particular properties.
+
 This an OOP implementation of Conway's game of life in C++. 
 Use of abstract classes and interface allows possibility of extension (inheriting from Board.h or NormalBoard.h) like:
- - Langton’s Ant
- - Brian’s Brain
- - Day and Night
- - etc..
+ - [Langton’s Ant](https://en.wikipedia.org/wiki/Brian%27s_Brain)
```
Comment (48cfu) [inline README.md]: wrong link

**Review by 48cfu — CHANGES_REQUESTED** [review]

**Review by 48cfu — APPROVED** [review]
thanks a lot

**Review by 48cfu — APPROVED** [review]

**Review by 48cfu — CHANGES_REQUESTED** [review]
sorry I made a mistake. Please go through the comments

File: README.md (hunk 1 above)
Comment (Tawishi) [reply README.md:29]: What links? Please specify.

File: README.md [hunk 4]
```diff
@@ -4,15 +4,14 @@
 This an OOP implementation of Conway's game of life in C++. 
 Use of abstract classes and interface allows possibility of extension (inheriting from Board.h or NormalBoard.h) like:
  - [Langton’s Ant](https://en.wikipedia.org/wiki/Brian%27s_Brain)
- - [Brian’s Brain](https://en.wikipedia.org/wiki/Brian%27s_Brain)
+ - [Brian’s Brain](https://en.wikipedia.org/wiki/Langton%27s_ant)
```
Comment (48cfu) [inline README.md]: Still mixed up. Langton's Ant points to Brian's brain and viceversa

**Review by 48cfu — CHANGES_REQUESTED** [review]

**Review by 48cfu — APPROVED** [review]
Thanks a lot.

**Comment by Tawishi** [comment]
Thank you for this opportunity.

**Pull request merged**